### ISSUE TITLE
Cannot parse tsconfig when using the relative path of tsconfig

### ISSUE THREAD

**Issue by brandonxiang** [pzavolinsky/ts-unused-exports#248]
```
./bin/ts-unused-exports example/simple/tsconfig.json
```

It throws error. `Cannot parse 'example/simple/tsconfig.json'.`

```
cd example/simple
./bin/ts-unused-exports ./tsconfig.json
```

It works well. Some bug in the relative path.

**Comment by mrseanryan** [comment]
thank you for reporting 

sean

**Comment by mrseanryan** [comment]
Released in 8.0.5
